### DISCUSSION TITLE
Is test mode gone?

### DISCUSSION THREAD

**Discussion by tmblue** [forkineye/ESPixelStick#425] [Q&A]
Don't see it in the www directory and don't see it anywhere on the stick, #test does nothing, stays in the current page. But since I'm not seeing it in the www tree, is it gone, or how does one get to it now?

**Comment by forkineye** [comment]
The former test modes are secondary inputs under effects in 4.0 and you can set them there.  More generic testing will be coming to the diagnostics tab in the future.
